trdg/handwritten_text_generator.py

```diff
@@ -10,6 +10,20 @@
 from PIL import Image, ImageColor
 from collections import namedtuple
 
+def download_model_weights():
+    from pathlib import Path
+    import urllib.request 
+    cwd = os.path.dirname(os.path.abspath(__file__))
+    for k in ['model-29.data-00000-of-00001','model-29.index','model-29.meta','translation.pkl']:
+        download_dir = Path(cwd)/'handwritten_model/'
+        download_dir.mkdir(exist_ok=True,parents=True)
+        if (download_dir/f'{k}').exists(): continue
+        print(f'file {k} not found, downloading from git repo..')
+        urllib.request.urlretrieve(
+            f'https://raw.github.com/Belval/TextRecognitionDataGenerator/master/trdg/handwritten_model/{k}', 
```

Comment:
Use
```
f'https://raw.github.com/Belval/TextRecognitionDataGenerator/7cf75c10bb8aa1c7e466a670eb471c1b0e53cc20/trdg/handwritten_model/{k}',
```
instead since it will be bound to the commit hash instead of HEAD.